packages/models/src/error.rs

```diff
@@ -23,6 +42,8 @@ pub enum ApiError {
     Unauthorized,
 
     NotFound,
+    Unknown(String),
+    BadRequest,
```

Comment:
Why do you use separated Error type? ApiError and Error. 
I recommend you to use same Error type for consistency.